core/src/main/java/bisq/core/dao/state/model/DaoState.java

```diff
@@ -109,9 +111,9 @@ public static DaoState getClone(DaoState daoState) {
     @JsonExclude
     private transient final Map<String, Tx> txCache; // key is txId
     @JsonExclude
-    private transient final Map<Integer, Block> blockHeightCache; // key is block height
+    private transient final Map<Integer, Block> blocksByHeight; // Blocks indexed by height
     @JsonExclude
-    private transient final Set<String> blockHashCache;
+    private transient final Set<String> blocksByHash; // Blocks indexed by hash
```

Comment:
With blocksByHash I would expect a map with hash as key and blocks as value. For a set I would use 'blockHashes'

Reply:
Agree. I had forgotten one is a set, the other a map.